package.json

```diff
@@ -209,7 +212,7 @@
   },
   "husky": {
```

Comment:
I believe that we don't need the husky config to be in package.json file anymore. It already is on  `.husky/` folder in the root directory.

Reply:
in fact it is no longer necessary, that was a leftover.